README.md

```diff
@@ -1,7 +1,13 @@
-How to run me:
+# How to run me:
 1. Clone repo
 2. Download the .env file from the Google Drive folder: 'Furlough Hackathon 2020' and place it into this project
 3. If this is your first time running the project, open an intergrated terminal and run the command: 'npm install'
 4. Open an intergrated terminal and enter the command: 'npm start'
-5. Run serveo and get your localhost linked to a public domain by entering the command: 'ssh -R team-awesome:80:localhost:3000 serveo.net'
-6. Log into Slack, Team-Awesome workspace, and starting talking to Reginald in the Apps section!
\ No newline at end of file
+5. Run serveo and get your localhost linked to a public domain by entering the command: 'ssh -R team-awesome:80:localhost:3000 serveo.net' or 'ssh -R 80:localhost:3000 ssh.localhost.run' (recommended)
+6. Log into Slack, Team-Awesome workspace, and starting talking to Reginald in the Apps section!
+
+# Slack Events and Commands configuration update
+To run Slack integration with your local setup, configure your Slack App as follows:
+1. Change the Event Subscriptions > Request URL to the one supplioed by Serveo or Localhost.run e.g. https://pjeyakumar-ff9a78cc.localhost.run/slack/events (hit the Save button)
```

Comment:
there is a type "supplioed"

Reply:
you can also replace my personal public url with the common one https://and-reginald-e6852d81.localhost.run